express-app/routes/booking.js

```diff
@@ -13,6 +13,10 @@ const Room = require('../models/room');
 // error handler
 const handleError = require('./errorHandler');
 
+// user auth middleware
+const authUser = require('../middleware/authUser.js');
+router.use('/create', authUser);
```

Comment:
why is the name of this route `/create`?

Reply:
as far as the code base is being developed, `/booking/create` is the only route that requires user auth. so the middleware is passed to that, and nothing else